nautobot/extras/plugins/urls.py

```diff
@@ -8,15 +8,23 @@
 from . import views
 
 # Initialize URL base, API, and admin URL patterns for plugins
-plugin_patterns = [
-    path("installed-plugins/", views.InstalledPluginsView.as_view(), name="plugins_list"),
-    path("installed-plugins/<str:plugin>/", views.InstalledPluginDetailView.as_view(), name="plugin_detail"),
+apps_patterns = [
+    path("installed-apps/", views.InstalledAppsView.as_view(), name="apps_list"),
+    path("installed-apps/<str:app>/", views.InstalledAppDetailView.as_view(), name="app_detail"),
+]
+plugin_patterns = [  # 3.0 TODO: remove these
+    path("installed-plugins/", views.InstalledAppsView.as_view(), name="plugins_list"),
+    path("installed-plugins/<str:plugin>/", views.InstalledAppDetailView.as_view(), name="plugin_detail"),
```

Comment:
Need to fix this legacy URL/view - get `TypeError: get() got an unexpected keyword argument 'plugin'` when accessing it.